JulioC-19/LargeProjectMobile
# Julio/image carousel

Implemented image carousel, parsing of ingredients in search screen, active icon color in bottom nav bar. 

To test:
Go to signup screen and try to create a new account, go thru the image carousel and select profile image. 
Verify email and log into the app, go to profile screen to verify the image is there. 
Verify the active icon color is goldenrod in nav bar. 
Verify ingredients are part of the recipe screen when searching for meals.
